src/lib/utils/index.ts

```diff
@@ -37,10 +37,6 @@ export {
 export { addAbsolutePosition } from './hits-absolute-position';
 export { addQueryID } from './hits-query-id';
 export { default as isFacetRefined } from './isFacetRefined';
-export { default as createSendEventForFacet } from './createSendEventForFacet';
-export {
-  createSendEventForHits,
-  createBindEventForHits,
-  SendEventForHits,
```

Comment:
This had an error coming from rollup (probably).
```
40: export {
41:   createSendEventForFacet,
42:   SendEventForFacet,
      ^
43: } from './createSendEventForFacet';
44: export * from './createSendEventForHits';
Error: 'SendEventForFacet' is not exported by src/lib/utils/createSendEventForFacet.ts
    at error (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:5330:30)
    at Module.error (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:9733:16)
    at handleMissingExport (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:9634:28)
    at Module.getVariableForExportName (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:9868:24)
    at /Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:9850:39
    at Array.map (<anonymous>)
    at Module.getTransitiveDependencies (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:9850:14)
    at Chunk$1.link (/Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:10894:48)
    at /Users/eunjaelee/workspace/instantsearch.js/node_modules/rollup/dist/rollup.js:13205:23
    at runMicrotasks (<anonymous>)
```

related to: https://github.com/rollup/plugins/issues/71

couldn't find a fix other than changing it to like here.

Reply:
the possible solution would be `export * from ...`. Did you do that?

Reply:
Yeah, `export * from ...` is the current status in this branch, and it's the only solution at the moment.